.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -8,13 +8,13 @@ labels: "type : bug"
 ## Issue
 
 Describe the issue you are facing. Show us the implementation: screenshots, gif, etc.
- 
+
 ## Expected
 
 Describe what should be the correct behaviour.
- 
+
 ## Steps to reproduce
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
```

Comment:
Should we keep those spaces? They can be handy in the case someone reuses these bullets (faster to start writing stuff). 
Though I should admit that 100% of the time I would just select this sample text and start writing over it ^^